docs/source/advanced/mixed_precision.rst

```diff
@@ -61,23 +166,51 @@ It is also possible to use BFloat16 mixed precision on the CPU, relying on MKLDN
 
     Trainer(precision="bf16")
 
-NVIDIA APEX Mixed Precision
----------------------------
 
-.. warning::
-
-    We strongly recommend to use the above native mixed precision rather than NVIDIA APEX unless you require more finer control.
+****************
+Single Precision
+****************
 
-`NVIDIA APEX <https://github.com/NVIDIA/apex>`__ offers some additional flexibility in setting mixed precision. This can be useful for when wanting to try out different precision configurations, such as keeping most of your weights in FP16 as well as running computation in FP16.
+PyTorch models train with 32-bit floating-point (FP32) arithmetic by default.
+Lightning uses 32-bit by default. You can also set it using:
 
 .. testcode::
-    :skipif: not _APEX_AVAILABLE or not torch.cuda.is_available()
 
-    Trainer(gpus=1, amp_backend="apex")
+    Trainer(precision=32)
 
-Set the `NVIDIA optimization level <https://nvidia.github.io/apex/amp.html#opt-levels>`__ via the trainer.
+
+****************
+Double Precision
+****************
+
+Lightning supports training models with double precision/64-bit. You can set it using:
 
 .. testcode::
-    :skipif: not _APEX_AVAILABLE or not torch.cuda.is_available()
 
-    Trainer(gpus=1, amp_backend="apex", amp_level="O2")
+    Trainer(precision=64)
+
+.. note::
+
+    Since in deep learning, memory is always a bottleneck, especially when dealing with a large volume of data and using a GPU with limited memory.
+    It is recommended using single precision for better speed with the price of a possible negligible loss of precision.
```

Comment:
Might be worth to talk 8-bit optimizer and lower can be used from third parties libraries such as https://github.com/facebookresearch/bitsandbytes even if Lightning doesn't provide anything out of the box yet.

Reply:
done!